__tests__/tests.js

```diff
@@ -39,3 +40,85 @@ test("no console errors and warnings", async () => {
 
   expect(errors).toHaveLength(0);
 });
+
+test("every city is in the toggle", async () => {
+  const data = fs.readFileSync("data/cities-polygons.geojson");
+  const expectedCities = JSON.parse(data)["features"].map(
+    (entry) => entry["properties"]["Name"]
+  );
```

Comment:
FYI `map` iterates over every single element in the array and applies the function to it. It's a more concise way of writing a for loop that creates a new array.